shell/platform/linux/fl_texture_registrar_test.cc

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_texture_registrar.h"
+#include "flutter/shell/platform/embedder/embedder.h"
+#include "flutter/shell/platform/linux/fl_external_texture_gl.h"
+#include "flutter/shell/platform/linux/fl_texture_registrar_private.h"
+#include "flutter/shell/platform/linux/testing/fl_test.h"
+#include "gtest/gtest.h"
+
+#include <gmodule.h>
+
+// Implements eglGetProcAddress but returns dummy functions.
+// static void* eglGetProcAddress(const char* name) {
```

Comment:
There's a bunch of commented out code in this PR, which isn't something we would land. I'm not clear what the status here is; is finishing the tests blocked on something? Your request for re-review didn't mention anything.